test/python/WMCore_t/Storage_t/SiteLocalConfig_t.py

```diff
@@ -156,6 +159,37 @@ def testLoadingConfigFromOverridenEnvVarriable(self):
                          "Error: Wrong site name.")
 
 
+    @attr("integration")
+    def testSlcPhedexNodesEqualPhedexApiNodes(self):
+        """
+        For each site, verify that the stageout node specified in
+        site-local-config.xml is the same as the one returned by the PhEDEx api.
+        """
+        os.environ["CMS_PATH"] = "/cvmfs/cms.cern.ch"
+
+        phedex = PhEDEx()
+        nodes = phedex.getNodeMap()["phedex"]["node"]
+
+        # Make a dict for translating the se names into regular site names.
+        node_map = {}
+        for node in nodes:
+            node_map[str(node[u"se"])] = str(node[str(u"name")])
+        
+        for d in os.listdir("/cvmfs/cms.cern.ch/SITECONF/"):
+            # Only T0_, T1_... folders are needed
+            if d[0] == "T":
+                os.environ['WMAGENT_SITE_CONFIG_OVERRIDE'] ='/cvmfs/cms.cern.ch/SITECONF/%s/JobConfig/site-local-config.xml' % (d)
+                try:
+                    slc = loadSiteLocalConfig()
+                except Exception as e:
```

Comment:
Hi Eric,

my intention with this is to use it as an `integration` unit test (run by hand) to discover faulty/broken SLC at sites (see the email I sent yesterday to the site support team). In that sense I would even improve the number of printouts/type or defect discovered by the unit test (I would also like to know which sites have not set the phedex-node attribute in their SLC).

I know these things belongs to the SST and a SAM test would be more appropriate, but imho the codebase should stay here since WMAgent and CRAB3 uses the WMCore code at runtime, and it makes also sense to test the code run by them. Maybe I can talk to Ali next week and see if they can recycle the unit test in some way?